tests/utils/configupdater/postgres/postgres.go

```diff
@@ -84,11 +85,16 @@ func (r *ConfigUpdater) CreateTrigger(channel string) error {
 }
 
 func (r *ConfigUpdater) Init(props map[string]string) error {
+	connString, _ := metadata.GetMetadataProperty(props, "connectionString")
```

Comment:
Here the use of GetMetadataProperty and IsTruthy is ok :)

Reply:
maybe this function should have a Dos / Don'ts comment in the docs so others know when to and to not use this if you feel so strongly about it

Reply:
Oh I think I can do one better -- I can probably add a custom linter which will now allow the use of this function for added code :)